docs/developer-guide/project-creation-script.md

```diff
@@ -69,3 +69,18 @@ In addition to static and templates, the following files from the root MapStore
 * .eslintrc
 * .eslintignore
 * LICENSE.txt
+
+## Update MapStore2 version in a project
+
+To update MapStore2 version enter in the MapStore2 folder and pull desired git version.
+If MapStore2 devDependencies have been changed you can manually update these in the project package.json file or run the script updateDevDeps
+```sh
+npm run ./updateDevDeps.js
```

Comment:
maybe node ./updateDevDeps.js ?

Reply:
No I have added as a script npm updateDevDeps is enough

Reply:
ok, but then it should be npm run updateDevDeps I suppose, not npm run ./updateDevDeps.js

Reply:
Yes, you are right and I have fixed ;-)